Update documentation for arrays that semantic field cannot support it (#11482)
opensearch-project/documentation-website: _mappings/supported-field-types/index.md

}
```

> semantic field cannot support an array of values since we will map it to an embedding field as rank_features/knn_vector, and they cannot support multiple vectors.

Reviewer: ```suggestion
The `semantic` field cannot contain an array of values because it's mapped to an embedding field (`rank_features` or `knn_vector`), which supports only a single vector.
```
